ts/components/conversation/Linkify.tsx

```diff
@@ -9,31 +9,314 @@ import { RenderTextCallbackType } from '../../types/Util';
 import { isLinkSneaky } from '../../../js/modules/link_previews';
 
 const linkify = LinkifyIt()
-  // This is all of the TLDs in place in 2010, according to [Wikipedia][0]. Note that
-  //   this only applies to "fuzzy" matches (`example.com`), not matches with
-  //   protocols (`https://example.com`).
-  // [0]: https://en.wikipedia.org/wiki/Generic_top-level_domain#History
+  // This is all TLDs in place in 2010, according to [IANA's root zone database][0].
```

Comment:
Given that we're planning to remove test TLDs, it could be worth calling that out here.

Reply:
You're right, I mentioned it in the new version.